CHANGELOG.md

```diff
@@ -3,6 +3,48 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.14.0
+
+_Jul 21, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧪 Export previously accessible components
+- 🌍 Different locales improvements
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@5.14.0` / `@mui/x-data-grid-pro@5.14.0` / `@mui/x-data-grid-premium@5.14.0`
+
+#### Changes
+
+- [DataGrid] Add generics to `GridPreProcessEditCellProps` (#5510) @YunosukeY
+- [DataGrid] Avoid inconsistent state export (#5390) @flaviendelangle
+- [DataGrid] Export `GridBooleanCell` component (#5537) @cliedeman
+- [DataGrid] Fix `date`/`dateTime` edit input font size to match view mode (#5304) @cherniavskii
+- [DataGrid] Fix grid loading overlays position (#5558) @DanailH
+- [DataGrid] Improve accessibility of Export menu in the toolbar (#5486) @nogalpaulina
+- [DataGridPremium] Implement Aggregation (#4208) @flaviendelangle
```

Comment:
I don't know why it did not take my last PR title
https://github.com/mui/mui-x/pull/4208

```suggestion
- [DataGridPremium] Implement Aggregation - not publicly released (#4208) @flaviendelangle
```